src/components/views/messages/MFileBody.js

```diff
@@ -372,14 +374,63 @@ module.exports = React.createClass({
                 </span>
             );
         } else if (contentUrl) {
+            const downloadProps = {
+                target: "_blank",
```

Comment:
I don't think `target` or `rel` apply to the fetch path? Or do they?

Reply:
`downloadProps` isn't related to `fetch`, but we weren't setting `rel` on one of the anchors later on, so this seems like a good place to make sure we don't forget it.